test-helpers/S3.ts

```diff
@@ -0,0 +1,20 @@
+import { S3Event, S3EventRecord } from "aws-lambda";
+
+export const createEvent = (records: S3EventRecord[]): S3Event => ({
+  Records: records,
+});
+
+export const createS3EventRecord = (
+  bucketName: string,
+  fileName: string
+): S3EventRecord =>
+  ({
+    s3: {
+      bucket: {
+        name: bucketName,
+      },
+      object: {
+        key: fileName,
+      },
+    },
+  } as any);
```

Comment:
Why `as any`? Is this not conforming to the `S3EventRecord` interface?

Reply:
It is `as any` because `s3` is one of many keys in the **interface S3EventRecord** and it isn't typed. Appreciate why it is confusing. Shall I just change the return type to be `any` instead of `S3EventRecord`?